drape_frontend/apply_feature_functors.cpp

```diff
@@ -412,6 +412,7 @@ void BaseApplyFeature::ExtractCaptionParams(CaptionDefProto const * primaryProto
   titleDecl.m_primaryTextFont = decl;
   titleDecl.m_primaryOffset = GetOffset(primaryProto->offset_x(), primaryProto->offset_y());
   titleDecl.m_primaryOptional = primaryProto->is_optional();
+  // Secondary captions are optional always.
```

Comment:
What are the benefits of making them always optional? What are the cons?

Reply:
I see no cons.

Even in the only case of a mandatory caption plus a star icon for capitals - if e.g. a country name displaces the secondary int_name of the capital then the primary name and a star icon better stay, right?

Other cases are like a train station with a secondary int_name. Also better to displace just the secondary one and let the primary stay instead of removing them both at once.